### DISCUSSION TITLE
How do I prevent my plate name from appearing at the top of the screen?

### DISCUSSION THREAD

**Discussion by spetryjohnson** [HASwitchPlate/openHASP#642] [Q&A] [answered]
OpenHASP newbie here.

I've been following examples and working my way through a config. I have both a Seeed SenseCAP D1 and a CYD, and they both show the plate name at the top of the screen.

None of the screenshots I've seen in the examples include this, but I can't see a setting to turn it off.

Am I missing something stupid?

![image](https://github.com/HASwitchPlate/openHASP/assets/275393/e7744792-07ec-4abe-b184-dc992b07638c)

**Comment by fvanroie** [comment, marked as answer]
What is the contents of the `pages.jsonl` file in the root of the device filesystem?

**Reply by spetryjohnson** [reply]
That was it! I didn't realize there was a file on the device filesystem itself, in addition to the files I was managing through Home Assistant.

Thanks!
